script/tasks/diamondSyncSigs_FAST.sh

```
@@ -0,0 +1,188 @@
+          else
+            printf '\033[0;32m%s\033[0m\n' "✅ [$NETWORK] Signatures synced"
+          fi
+        else
+          printf '\033[0;32m%s\033[0m\n' "✅ [$NETWORK] Signatures already approved"
+        fi
+        return
+      fi
+
+      # Fallback to cast send
+      if [[ $ATTEMPTS == 1 ]]; then
+        CFG_SIGS=($(jq -r '.[] | @sh' "./config/sigs.json" | tr -d \' | tr '[:upper:]' '[:lower:]'))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)